src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.AcceptAllCerts.cs

```diff
@@ -57,9 +60,8 @@ public void SingletonReturnsTrue()
             new object[] { Configuration.Http.WrongHostNameCertRemoteServer },
         };
 
-        [ActiveIssue(7812, TestPlatforms.Windows)]
         [OuterLoop] // TODO: Issue #11345
-        [Theory]
+        [ConditionalTheory(nameof(ClientSupportsDHECipherSuites))]
```

Comment:
Please refer to #7812.  This is prevent CI failures.  We don't want these tests running on Windows OS versions that don't have the fixes to the DHE handshakes bugs.  It is not a feature we want to test, per se.